source/extensions/tracers/lightstep/lightstep_tracer_impl.h

```diff
@@ -60,6 +60,8 @@ class LightStepDriver : public Common::Ot::OpenTracingDriver {
   Runtime::Loader& runtime() { return runtime_; }
   LightstepTracerStats& tracerStats() { return tracer_stats_; }
 
+  static const size_t DEFAULT_MIN_FLUSH_SPANS;
```

Comment:
Nit: Prefer `DefaultMinFlushSpans` for Envoy recommended constant style.

Reply:
Renamed.